src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html

```diff
@@ -27,6 +27,10 @@
 <H2>Expression Language Injection</H2>
 The software constructs all or part of an expression language (EL) statement in a Java Server Page (JSP) using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended EL statement before it is executed. In certain versions of Spring 3.0.5 and earlier, there was a vulnerability (CVE-2011-2730) in which Expression Language tags would be evaluated twice, which effectively exposed any application to EL injection. However, even for later versions, this weakness is still possible depending on configuration.
 
+<H2>FormatString</H2>
```

Comment:
This is better if it exactly matches the name of the alert ("Format String" or "Format String Error", see above comment).